examples/persistence-service-catalog.md

```diff
@@ -16,19 +16,18 @@ sudo curl -L https://github.com/snowdrop/spring-boot-cloud-devex/releases/downlo
 sudo chmod +x /usr/local/bin/sb
 ```
 
-- odo's tool to create the service from the catalog
+- odo which will provide the ability to run the spring boot application on Openshift and also create the PostgreSQL database
 
 ```bash
 cd $GOPATH/src/github.com/redhat-developer
 git clone https://github.com/redhat-developer/odo.git && cd odo
-git fetch origin pull/622/head:pr-723
-git checkout pr-723
 make install && sudo cp $GOPATH/bin/odo /usr/local/bin
 ```
 
 - Create a Minishift's vm running okd 3.10 and service catalog 
 
 ```bash
+minishift addons enable xpaas
```

Comment:
Could you add also `minishift addons enable admin-user` ?

Reply:
Good idea, yes